app/src/features/projects/components/ProjectFundingItemForm.test.tsx

```diff
@@ -0,0 +1,168 @@
+import { IMultiAutocompleteFieldOption } from 'components/fields/MultiAutocompleteFieldVariableSize';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import renderer from 'react-test-renderer';
+import { IInvestmentActionCategoryOption } from './ProjectFundingForm';
+import ProjectFundingItemForm, {
+  IProjectFundingFormArrayItem,
+  ProjectFundingFormArrayItemInitialValues
+} from './ProjectFundingItemForm';
+
+const funding_sources: IMultiAutocompleteFieldOption[] = [
+  {
+    value: 1,
+    label: 'agency 1'
+  },
+  {
+    value: 2,
+    label: 'agency 2'
+  },
+  {
+    value: 3,
+    label: 'agency 3'
+  }
+];
+
+const investment_action_category: IInvestmentActionCategoryOption[] = [
+  {
+    value: 1,
+    fs_id: 1,
+    label: 'action 1'
+  },
+  {
+    value: 2,
+    fs_id: 2,
+    label: 'category 1'
+  },
+  {
+    value: 3,
+    fs_id: 3,
+    label: 'not applicable'
+  }
+];
+
+describe.skip('ProjectFundingItemForm', () => {
```

Comment:
remove the skip?

Reply:
This test is broken at the moment.  There is some issue with modals that I havent figured out how to address.  I actually didnt meant to commit that file, but plan to fix it before any of this gets merged regardless